phpstan.neon

```diff
@@ -157,7 +157,6 @@ parameters:
     - '#Property App\\Models\\Payment::\$(?:amount|processing_fee) \(string(?:\|null)?\) does not accept (?:float|int|float\|int)\.#'
     - '#Property App\\Models\\Signature::\$user_agent \(string\|null\) does not accept array\|string\|null\.#'
     - '#Property Spatie\\Permission\\Contracts\\(Permission|Role)::\$name \(string\) does not accept mixed\.#'
-    - '#Regex pattern is invalid: Unknown modifier ''@'' in pattern: .+#'
```

Comment:
I should probably look more closely at what you're ignoring in the future

Reply:
Well also phpstan is wrong a lot so if i don't already know it's correct I'll probably ignore it